src/hmrc-content-guide/recognised-overseas-pension-schemes-notification-list/index.njk

```diff
@@ -0,0 +1,39 @@
+---
+title: Recognised overseas pension schemes notification list
+menuText: Recognised overseas pension schemes notification list
+layout: twoColumnPage.njk
+---
+{% from "_example.njk" import example %}
+
+<h2 class="govuk-heading-l">Document type</h2>
```

Comment:
Is the H1 added in the example.njk? Can't remember without looking at the nunjucks files.

Reply:
Yep, the H1 for this is 'Recognised overseas pension schemes notification list'